### PR TITLE
Adding navs to book store 

### PR DESCRIPTION
**Pull request by alexander16108**
- In this, we add navigation.

- We made the page replicate a SPA (single page Application)

- We Made sure that when you click on any of the links, the main area changes to display only one of the three content sections. 

- We Displayed the current date below the navigation bar.

### CODE REVIEW

**Review by de-scholar — CHANGES_REQUESTED** [review]
Hi @alexander16108 and @chaw-bot , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a Pull Request lacking description

Your Code Review Request will be marked as invalid in your Dashboard, so please add clear description on your Pull Request and re-submit a new code review request once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: js/nav.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  addSec.style.display = 'none';
+  contactSec.style.display = 'none';
+}
+
+// display add section
+function displayAdd() {
+  listSec.style.display = 'none';
+  addSec.style.display = 'block';
+  contactSec.style.display = 'none';
+}
```
Comment (Georjane) [inline js/nav.js:20]: - [x] Your design looks great! But I think in an effort to add the navbar functionality, you broke something in your code. Right now when I add a book, I don't see the book on the booklist when I click on the list. I only see the books I added when I refresh the page. I think you can add this functionality to your code by using `window.reload()` after a book is added or any other method you think can fix this issue.

File: README.md [hunk 2]
```diff
@@ -5,20 +5,26 @@
...
+
+### Live Version Link
+
+ [Live Version](https://alexander16108.github.io/Chaws-Awesome-B00k-Store/)
+
 
 ### Build With
  - HTML
  - JAVASCRIPT
  - CSS
```
Comment (Georjane) [inline README.md:27]: ## Optional suggestions

- [ ] You can add steps for setting up your project locally here.

**Review by Georjane — CHANGES_REQUESTED** [review]
### Hello, Team <STUDENTS>, ✋🏽 

Good job so far! You have a very professional Readme file, your PR is professional and you have a good mastery of git-flow. Your linters are all passing, I applaud that too.
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding steps for setting up your project locally in your Readme file.

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: js/nav.js (hunk 1 above)
Comment (alexander16108) [reply js/nav.js:20]: Thank you we will work on that right now

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi  Team 
## STATUS APPROVED ✔️ 
Great job implementing the changes requested by the previous reviewer 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**